Emoji.Wpf/Emoji.Wpf.csproj

```diff
@@ -21,111 +17,53 @@
     </Task>
   </UsingTask>
   <PropertyGroup>
-    <Configuration Condition="'$(Configuration)'==''">Release</Configuration>
-    <Platform Condition="'$(Platform)'==''">AnyCPU</Platform>
-    <ProjectGuid>{6A9C0AD6-14F7-451A-A0D4-38987A275F0D}</ProjectGuid>
+  <Title>Emoji.Wpf</Title>
+  <Description>Emoji rendering for WPF applications</Description>
+  <Product>Emoji.Wpf</Product>
+  <Copyright>Copyright © 2017—2021 Sam Hocevar</Copyright>
     <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>Emoji.Wpf</RootNamespace>
-    <AssemblyName>Emoji.Wpf</AssemblyName>
-    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
-    <TargetFrameworkProfile />
+    <TargetFrameworks>net40;netcoreapp3.1;net50-windows</TargetFrameworks>
+    <GenerateAssemblyFileVersionAttribute>false</GenerateAssemblyFileVersionAttribute>
+  <GenerateAssemblyInformationalVersionAttribute>false</GenerateAssemblyInformationalVersionAttribute>
+  <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
```

Comment:
Why these properties? The `AssemblyInfo.cs` file has been removed. Usually you set these when you want to use attributes in that file instead.

Reply:
This is necessary because these attributes are already generated by GitVersion.

Reply:
I would expect GitVersion to do this suppression, in that case.

Reply:
GitVersion was conflicting, so these lines had to be there